### PR TITLE
Feat: Modal component

### PR DESCRIPTION
**Pull request by Muzammil-khan710**
added modal component

### CODE REVIEW

File: docs/components/Modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
+
+.modal-container {
+    border: 1px solid black;
```
Comment (Shubhamkr2610) [inline docs/components/Modal/modal.css]: suggestion: you should use color with variable name  to maintain the consistency with naming convention

**Review by Shubhamkr2610 — APPROVED** [review]
Code is clean and easy to understand . You can merge it

File: docs/components/Modal/modal.css (hunk 1 above)
Comment (Muzammil-khan710) [reply docs/components/Modal/modal.css]: ### sure bro

**Review by Shubhamkr2610 — APPROVED** [review]

**Review by developer-anand — APPROVED** [review]
Clean code. Can be merged.

**Pull request merged**